### PR TITLE
Added mobile view for lobby

### PR DESCRIPTION
**Pull request by esme**
- moved theme background and colors to app.tsx
- refactored lobby code to include lobbyMobile and lobbyDesktop components
- used React Hooks to display component in mobile view

### CODE REVIEW

File: packages/frontend/store/system/types.ts [hunk 1]
```diff
@@ -1,14 +1,29 @@
 export const SET_MOBILE_VIEW = 'SET_MOBILE_VIEW';
+export const SET_WINDOW_DIMENSIONS = 'SET_WINDOW_DIMENSIONS';
 
 export type MobileView = boolean;
+export type Width = number;
+export type Height = number;
+export type WindowDimensions = {
```
Comment (vck3000) [inline packages/frontend/store/system/types.ts]: Let's get rid of Width and Height individual declarations and inside WindowDimensions do the following:
```
width: number;
height: number;
```

File: packages/frontend/store/system/actions.ts [hunk 2]
```diff
@@ -1,14 +1,28 @@
...
   return {
     type: SET_MOBILE_VIEW,
     mobileView,
   };
 };
 
-export default setMobileView;
+export const setWindowDimensions = (
+  width: Width,
+  height: Height,
```
Comment (vck3000) [inline packages/frontend/store/system/actions.ts]: Let's given in a WindowDimension object here instead of two separate numbers.

Comment (esme) [reply packages/frontend/store/system/actions.ts]: What if we need the type for width or height one day? Should we do setWindowDimensions.width?

Comment (vck3000) [reply packages/frontend/store/system/actions.ts]: image
Can do something like this :P

File: packages/frontend/pages/lobby.tsx [hunk 3]
```diff
@@ -14,138 +18,325 @@ import Chat, { dateGenObj } from '../components/lobby/chat';
...
+  mobileView,
+  theme,
+  windowDimensions,
+}: IStateProps): ReactElement => (
+  <>
+    <title>Lobby</title>
+    {mobileView ? (
+      <LobbyMobile theme={theme} windowDimensions={windowDimensions} />
+    ) : (
+      <LobbyDesktop />
```
Comment (vck3000) [inline packages/frontend/pages/lobby.tsx]: Is it possible to have LobbyMobile and LobbyDesktop in separate files? :)

Comment (esme) [reply packages/frontend/pages/lobby.tsx]: Yes should I put them in components/lobby?

**Comment by esme** [comment]
I'm getting console errors because React.fragment from <LobbyLeftPanel /> is nested inside <Grid></Grid>

**Comment by vck3000** [comment]
> I'm getting console errors because React.fragment from is nested inside

Resolved^. Needed to use `< />` instead of `{}` when inserting React Elements!

**Comment by vck3000** [comment]
Let's add the navbar and fix up the lobby page logo positioning for mobile before we merge this!

**Pull request merged**